testsuite/MDAnalysisTests/analysis/test_waterdynamics.py

```diff
@@ -22,64 +22,81 @@
-        hbl.run(verbose=False)
-        assert_equal(round(hbl.timeseries[2][1],5), 0.75)
-
-    def test_WaterOrientationalRelaxation(self):
-        wor = MDAnalysis.analysis.waterdynamics.WaterOrientationalRelaxation(self.universe, 
-                                                    self.selection1, 0, 5, 2)
-        wor.run(verbose=False)
-        assert_equal(round(wor.timeseries[1][2],5), 0.35887)
-
-    def test_WaterOrientationalRelaxation_zeroMolecules(self):
-        wor_zero = MDAnalysis.analysis.waterdynamics.WaterOrientationalRelaxation(self.universe, 
-                                                    self.selection3, 0, 5, 2)
-        wor_zero.run(verbose=False)
-        assert_equal(wor_zero.timeseries[1], (0.0, 0.0, 0.0))
-
-    def test_AngularDistribution(self):
-        ad = MDAnalysis.analysis.waterdynamics.AngularDistribution(self.universe, 
-                                                            self.selection1, 40)
-        ad.run(verbose=False)
-        assert_equal(str(ad.graph[0][39]), str("0.951172947884 0.48313682125") )
-
-    def test_MeanSquareDisplacement(self):
-        msd = MDAnalysis.analysis.waterdynamics.MeanSquareDisplacement(self.universe, 
-                                                    self.selection1, 0, 10, 2)
-        msd.run(verbose=False)
-        assert_equal(round(msd.timeseries[1],5), 0.03984)
-
-    def test_MeanSquareDisplacement_zeroMolecules(self):
-        msd_zero = MDAnalysis.analysis.waterdynamics.MeanSquareDisplacement(self.universe, 
-                                                    self.selection3, 0, 10, 2)
-        msd_zero.run(verbose=False)
-        assert_equal(msd_zero.timeseries[1], 0.0)
-
-    def test_SurvivalProbability(self):
-        sp = MDAnalysis.analysis.waterdynamics.SurvivalProbability(self.universe, 
-                                                    self.selection1, 0, 6, 3)
-        sp.run(verbose=False)
-        assert_equal(round(sp.timeseries[1],5), 1.0)
-    
-    def test_SurvivalProbability_zeroMolecules(self):
-        sp_zero = MDAnalysis.analysis.waterdynamics.SurvivalProbability(self.universe, 
-                                                    self.selection3, 0, 6, 3)
-        sp_zero.run(verbose=False)
-        assert_equal(sp_zero.timeseries[1], 0.0)
+SELECTION1 = "byres name OH2"
+SELECTION2 = SELECTION1
+SELECTION3 = "byres name P1"
+
+
+@pytest.fixture()
+def universe():
+    return MDAnalysis.Universe(waterPSF, waterDCD)
+
+
+def test_HydrogenBondLifetimes(universe):
+    hbl = MDAnalysis.analysis.waterdynamics.HydrogenBondLifetimes(universe,
+                                                                  SELECTION1,
+                                                                  SELECTION2,
+                                                                  0, 5, 3)
+    hbl.run(verbose=False)
+    assert round(hbl.timeseries[2][1], 5) == 0.75
```

Comment:
It looks like the test tries to compare `hbl.timeseries[2][1]` to 0.75 but has precision issues. This is what `assert_almost equal` is for.